server/pbenchinacan/etc/pbench-server/pbench-server.cfg

```diff
@@ -31,6 +31,8 @@ secret-key = "pbench-in-a-can secret shhh"
 [openid-connect]
 #server_url = http://localhost:8090
 secret = <keycloak_secret>
+realm = <keycloak_realm>
+client = <keycloak_client>
```

Comment:
@npalaska, what stuff should we have here?  (I've lost track...do we still need/want the `secret`?)

This is only used by the functional test deployment, but I'm assuming that we want the `server_url` left commented out until we merge the OIDC branch into main, right?

Reply:
We don't need a secret as we are creating a public client. 

> This is only used by the functional test deployment, but I'm assuming that we want the `server_url` left commented out until we merge the OIDC branch into main, right?

That is correct.

Reply:
OK, I've removed `secret` and left `server_url` commented out.